.drone.yml

```diff
@@ -233,5 +225,282 @@ services:
+  - end-to-end-tests-server
+
+- name: build-storybook
+  image: grafana/build-container:1.2.21
+  commands:
+  - rm -rf $(yarn cache dir) && cp -r yarn-cache $(yarn cache dir)
+  - yarn storybook:build
+  - echo $${GCP_KEY} > /tmp/gcpkey.json
+  - gcloud auth activate-service-account --key-file=/tmp/gcpkey.json
+  - gsutil -m rsync -d -r ./packages/grafana-ui/dist/storybook gs://grafana-storybook/canary
+  environment:
+    GCP_KEY:
+      from_secret: gcp_key
+  depends_on:
+  - package
+
+- name: build-docs-website
+  image: grafana/docs-base:latest
+  commands:
+  - mkdir -p /hugo/content/docs/grafana
+  - cp -r docs/sources /hugo/content/docs/grafana/latest
+  - cd /hugo && make prod
+  depends_on:
+  - initialize
+
+- name: copy-packages-for-docker
+  image: grafana/build-container:1.2.21
+  commands:
+  - cp dist/*.tar.gz packaging/docker/
+  depends_on:
+  - package
+
+- name: build-docker-images
+  image: grafana/drone-grafana-docker:0.2.0
+  settings:
+    dry_run: true
+    edition: oss
+  depends_on:
+  - copy-packages-for-docker
+
+- name: build-docker-images-ubuntu
+  image: grafana/drone-grafana-docker:0.2.0
+  settings:
+    dry_run: true
+    edition: oss
+    ubuntu: true
+  depends_on:
+  - copy-packages-for-docker
+
+- name: postgres-integration-tests
+  image: grafana/build-container:1.2.21
+  commands:
+  - apt-get update
+  - apt-get install -yq postgresql-client
+  - ./bin/dockerize -wait tcp://postgres:5432 -timeout 120s
+  - psql -p 5432 -h postgres -U grafanatest -d grafanatest -f devenv/docker/blocks/postgres_tests/setup.sql
+  - rm -rf $(go env GOCACHE) && cp -r go-cache $(go env GOCACHE)
+  - go clean -testcache
+  - ./bin/grabpl integration-tests --database postgres
+  environment:
+    GRAFANA_TEST_DB: postgres
+    PGPASSWORD: grafanatest
+    POSTGRES_HOST: postgres
+  depends_on:
+  - test-backend
+  - test-frontend
+
+- name: mysql-integration-tests
+  image: grafana/build-container:1.2.21
+  commands:
+  - apt-get update
+  - apt-get install -yq default-mysql-client
+  - ./bin/dockerize -wait tcp://mysql:3306 -timeout 120s
+  - cat devenv/docker/blocks/mysql_tests/setup.sql | mysql -h mysql -P 3306 -u root -prootpass
+  - rm -rf $(go env GOCACHE) && cp -r go-cache $(go env GOCACHE)
+  - go clean -testcache
+  - ./bin/grabpl integration-tests --database mysql
+  environment:
+    GRAFANA_TEST_DB: mysql
+    MYSQL_HOST: mysql
+  depends_on:
+  - test-backend
+  - test-frontend
+
+- name: build-windows-installer
+  image: grafana/wix-toolset-ci:v3
+  commands:
+  - echo ./scripts/build/ci-msi-build/ci-msi-build-oss.sh
+  depends_on:
+  - package
+
+- name: release-next-npm-packages
+  image: grafana/build-container:1.2.21
+  commands:
+  - npx lerna bootstrap
+  - echo //registry.npmjs.org/:_authToken=$${NPM_TOKEN} >> ~/.npmrc"
+  - echo ./scripts/circle-release-next-packages.sh
+  environment:
+    NPM_TOKEN:
+      from_secret: npm_token
+  depends_on:
+  - end-to-end-tests
+
+- name: publish-packages
```

Comment:
This is already done by CircleCI master build, correct? If so, need to be removed for now.

Reply:
@marefr No need to worry, the publishing is only simulated (see `echo` at line 475).